Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/AsyncPDFRenderer.h

```diff
@@ -63,20 +63,27 @@ using PDFTileRenderIdentifier = ObjectIdentifier<PDFTileRenderType>;
 
 struct TileRenderInfo {
     WebCore::FloatRect tileRect;
-    std::optional<WebCore::FloatRect> clipRect; // If set, represents the portion of the tile that needs repaint (in the same coordinate system as tileRect).
+    WebCore::FloatRect renderRect; // Represents the portion of the tile that needs rendering (in the same coordinate system as tileRect).
+    RefPtr<WebCore::NativeImage> background; // Optional existing content around renderRect.
```

Comment:
What is the geometry of this background? Is it assumed to be previous the tile contents? If so, please say in the comment.